sdk/src/gateway/client.ts

```diff
@@ -187,10 +196,19 @@ export class GatewayApiClient {
             },
         });
 
-        const quote: GatewayQuote = await response.json();
+        const jsonResponse = await response.json();
+
+        if (!jsonResponse.orderDetails) {
+            throw new Error('Missing orderDetails in quote response');
+        }
+
+        const quote: GatewayQuote = {
+            ...jsonResponse,
+            orderDetails: convertOrderDetailsRawToOrderDetails(jsonResponse.orderDetails),
```

Comment:
This is still open for discussion. My assumption, I would prefer that SDK users see bigint values rather than hexadecimal representations.